Tests/SwiftSyntaxBuilderTest/ExpressibleBuildablesTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+import SwiftSyntax
+
+import SwiftSyntaxBuilder
+
+final class ExpressibleBuildablesTests: XCTestCase {
+    func testExpressibleAsMemberDeclListItem() {
+        let myStruct = StructDecl(identifier: "MyStruct", members: MemberDeclBlock(membersBuilder: {
+            VariableDecl(.var, name: "myFirstVar", type: "Int")
+
+            VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
```

Comment:
Do you want to test that both the `VariableDecl` convenience initializer and the `bindingsBuilder` initializer work here? If so, could you add a comment explaining your intention?

Reply:
Removing it. cannot remember why.